server/cli/start.go

```diff
@@ -90,29 +89,29 @@ func runInit(cmd *commander.Command, args []string) {
 
 	err := Context.Init()
 	if err != nil {
-		log.Errorf("Failed to initialize context: %v", err)
+		log.Errorf("failed to initialize context: %v", err)
 		return
 	}
 	e := Context.Engines[0]
 	if _, ok := e.(*engine.InMem); ok {
-		log.Errorf("Cannot initialize a cluster using an in-memory store")
+		log.Errorf("cannot initialize a cluster using an in-memory store")
 		return
 	}
 	// Generate a new UUID for cluster ID and bootstrap the cluster.
 	clusterID := uuid.New()
 	stopper := util.NewStopper()
 	if _, err = server.BootstrapCluster(clusterID, e, stopper); err != nil {
-		log.Errorf("Failed to bootstrap cluster: %v", err)
+		log.Errorf("failed to bootstrap cluster: %v", err)
 		return
 	}
 	stopper.Stop()
 
-	fmt.Printf("Cockroach cluster %s has been initialized\n", clusterID)
-	if Context.BootstrapOnly {
-		fmt.Printf("To start the cluster, run \"cockroach start\"\n")
+	log.Infof("cockroach cluster %s has been initialized\n", clusterID)
```

Comment:
I'm trying to get rid of unnecessary capitalization in the logs. Makes parsing the logs more difficult because the caps stand out.